src/doc/en/installation/index.rst

```diff
@@ -4,219 +4,143 @@
-
-    - `Set up your Linux username and password.
-      <https://learn.microsoft.com/en-us/windows/wsl/setup/environment#set-up-your-linux-username-and-password>`_
-      Do not include any spaces in your username.
-
-    - If your computer has less than 8GB of RAM, `change the WSL settings
-      <https://learn.microsoft.com/en-us/windows/wsl/wsl-config#main-wsl-settings>`_
-      to make at least 4GB of RAM available to WSL.
-
-    Start Ubuntu from the Start menu, and type the following commands
-    to install Sage from conda-forge. (The ``$`` represents the command
-    line prompt, don't type it!) The second step will ask a few questions,
-    and you may need to hit :kbd:`Enter` to confirm or type ``yes``
-    and then hit :kbd:`Enter`.
-
-    .. code-block:: shell
-
-       $ curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
-       $ bash Miniforge3-$(uname)-$(uname -m).sh
-       $ conda create -n sage sage python=3.11
-
-    (If there are any installation failures, please report them to
-    the conda-forge maintainers by opening a `GitHub Issue for
-    conda-forge/sage-feedstock <https://github.com/conda-forge/sage-feedstock/issues>`_.)
-
-    You can now start SageMath as follows:
-
-    .. code-block:: shell
-
-       $ conda activate sage
-       $ sage
-
-    This way of starting Sage gives you the most basic way of using
-    Sage in the terminal. See :ref:`sec-launching` for recommended next steps,
-    in particular for setting up the Jupyter notebook, which is required if
-    you want to use graphics.
+**I don't want to do Sage development:** Then follow instructions in
+:ref:`sec-installation-conda` and :ref:`sec-installation-conda-binary`. This is
+the easiest way of using Sage in the terminal. Then see :ref:`sec-post-installation` for
+recommended next steps, in particular for setting up the Jupyter notebook,
+which is convenient if you want to use graphics.
 
 .. _installation-guide-linux:
 
 Linux
-=====
+-----
 
-- **Do you want to do SageMath development?**
+**I want to do Sage development:**
 
-  - **Yes, development:**
+- Obtain the Sage sources via ``git`` as described in `the Sage Developer Guide
+  <https://doc.sagemath.org/html/en/developer/walkthrough.html#chapter-walkthrough>`_.
 
-    Obtain the SageMath sources via ``git`` as described in `The Sage
-    Developer's Guide
-    <https://doc.sagemath.org/html/en/developer/walkthrough.html#chapter-walkthrough>`_.
+- Then build Sage from source, choosing ``develop`` git branch, as described in
+  section :ref:`sec-installation-from-sources`.
 
-    - Then build SageMath from source as described in section
-      :ref:`sec-installation-from-sources`.
+- Alternatively, follow the instructions in section
+  :ref:`sec-installation-conda-develop`;
+  these describe an experimental method that gets all required
```

Comment:
Conda is not experimental since a very long time.

Given the many (recent?) problems with installing various packages in sage-the-distro, conda should actually be mentioned first, and only then the installation from source. Would safe us a lot of headache.

Reply:
I don't agree. Traditionally building from source is the main way of installing sage for developers.

But perhaps this may be debatable in sage-devel. If there is a consensus in favor of conda, then we may change this part later. But this is for another PR.

Not in scope of this PR.

Reply:
Making conda the default has been already discussed on the mailing list with only positive feedback (by people other than me): https://groups.google.com/g/sage-devel/c/AvH3xq2bCfo/m/OKwqzrrYCwAJ

In either way, this mode of installation is definitely not "experimental".

Reply:
OK, I removed "experimental".

I don't mind whether conda is the default or not. I just don't want to make the changes here myself because I don't have much experience with conda and I am not sure if the change is desirable. 

I would be happy if you make a followup PR just for the change.